DeepSpeech.ipynb

```diff
@@ -1444,13 +1444,16 @@
     "            # Loading the model into the session\n",
     "            execution_context[3].restore(session, model_path)\n",
     "\n",
+    "        # Create Coordinator to manage threads\n",
+    "        coord = tf.train.Coordinator()\n",
+    "\n",
     "        # Start queue runner threads\n",
-    "        tf.train.start_queue_runners(sess=session)\n",
+    "        managed_threads = tf.train.start_queue_runners(sess=session, coord=coord)\n",
     "\n",
     "        # Start importer's queue threads\n",
-    "        execution_context[1].start_queue_threads(session)\n",
+    "        managed_threads = managed_threads + execution_context[1].start_queue_threads(session, coord)\n",
     "\n",
-    "    return session"
+    "    return session, coord, managed_threads"
```

Comment:
One could think of bundling `session, coord, managed_threads` into another tuple. Just a thought - it's ok like that.